docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md

```diff
@@ -37,7 +37,7 @@ Create a Data Asset to define the data you want GX Cloud to access. Currently, t
 
     - **Account identifier**: Enter your Snowflake account or locator information. The locator value must include the geographical region. For example, `us-east-1`. To locate these values see [Account Identifiers](https://docs.snowflake.com/en/user-guide/admin-account-identifier).
 
-    - **Password**: Enter your Snowflake password.
+    - **Password**: Enter the password for the Snowflake account you're connecting to GX Cloud. To improve data security, GX recommends using a Snowflake service account to connect to GX Cloud.
```

Comment:
```suggestion
    - **Password**: Enter the password for the Snowflake user you're connecting to GX Cloud. To improve data security, GX recommends using a Snowflake service account to connect to GX Cloud.
```

In the context of Snowfalke, `account` and `user` have very specific meanings. `account` is inappropriate here.